CHANGELOG.md

```diff
@@ -1,3 +1,350 @@
+## [Talos 0.12.0-alpha.1](https://github.com/talos-systems/talos/releases/tag/v0.12.0-alpha.1) (2021-08-13)
+
+Welcome to the v0.12.0-alpha.1 release of Talos!  
+*This is a pre-release of Talos*
+
+
+
+Please try out the release binaries and report any issues at
+https://github.com/talos-systems/talos/issues.
+
+### Support for Self-hosted Control Plane Dropped
+
+> **Note**: This item only applies to clusters bootstrapped with Talos <= 0.8.
+
+Talos 0.12 completely removes support for self-hosted Kubernetes control plane (bootkube-based).
+Talos 0.9 introduced support for Talos-managed control plane and provided migration path to convert self-hosted control plane
+to Talos-managed static pods.
+Automated and manual conversion process is available in Talos from 0.9.x to 0.11.x.
+For clusters bootstrapped with bootkube (Talos <= 0.8), please make sure control plane is converted to Talos-managed before
+before upgrading to Talos 0.12.
```

Comment:
before - 2 times

Reply:
thanks, I'll get it fixed for the next beta!